build/log.go

```diff
@@ -52,9 +52,9 @@ var logCompressors = map[string]string{
 	Zstd: "zst",
 }
 
-// SuportedLogCompressor returns whether or not logCompressor is a supported
+// supportedLogCompressor returns whether or not logCompressor is a supported
```

Comment:
I think these will be used in other projects (e.g. `tapd`), so we shouldn't un-export them.